lib/Migration/Version0024Date20220203123902.php

```diff
@@ -124,6 +124,11 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			$table->setPrimaryKey(['token', 'instance']);
 		}
 
+		if ($schema->hasTable('circles_membership')) {
```

Comment:
this needs to be moved in another migration :)

Reply:
fixed :]